lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -543,9 +557,53 @@ MITIGATION_02_CRI_DCMAW:
       targetState: PYI_NO_MATCH
```

Comment:
Should DCMAW not found also lead to F2F ?

Reply:
I don't think so no - I think we said that `not-found` represents this part of the figma diagram, therefore we should route to PYI_NO_MATCH
![image](https://github.com/govuk-one-login/ipv-core-back/assets/55875065/bd724c43-967c-460f-aab9-d393ffa1df87)